docs/io/configuration/components/models/converters/arepo_to_tardis.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>I would explain what is included in the JSON file/how you get the file.</p><p> </p><p>Also, see if you can link the specific section of the API, or just summarize the options (or use the python help function in the notebook, any of those work)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>